test/blurring-method.js

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('blurring method', function() {
+  const testCases = [
+    {
+      'description': 'jpg image',
+      'inputImageUrl': 'test-files/64kb.jpg',
+      'expectedImageUrl': 'blurred/test1.png',
```

Comment:
Why is there a change of extensions (jpg -> png) ?

Reply:
I get the expected image from canvas, which is used for blurring. When the image is got from canvas, the default extension is png, it doesn't know the extension of the image which was originally put on canvas.

Reply:
Hmm, that's a bit counter-intuitive. I remember we talked about giving the user a choice of saving the image in either jpeg or png and we decided not to do it. But this is different. If I upload an image in jpeg format I kind of expect it to be back in the same format. How difficult is it to maintain the original type of the image?

Reply:
I guess in this case we will just need to remember somewhere the original type of an image and then get the image of this type from canvas. Canvas has a method to get image from it with the parameter of image type, which defaults to png. I was planning to do that on download button PR, when I will actually need to get image of some type from canvas.

Reply:
Ah, all right. That sounds good then!